raster/src/main/scala/geotrellis/raster/crop/MultibandTileCropMethods.scala

```diff
@@ -28,26 +28,40 @@ trait MultibandTileCropMethods extends TileCropMethods[MultibandTile] {
   import Crop.Options
 
   /**
-    * Given a [[GridBounds]] and some cropping options, crop the
-    * [[MultibandTile]] and return a new MultibandTile.
+    * Given a [[GridBounds]], a sequence of bands indexes,  and some cropping options, crop the
+    * [[MultibandTile]] and return a new MultibandTile that contains the target area and bands.
     */
-  def crop(gb: GridBounds, options: Options): MultibandTile = {
+  def crop(gb: GridBounds, targetBands: Array[Int], options: Options): MultibandTile = {
```

Comment:
I suggest calling this `cropBands` to keep it consistent with `readSubsetBands` and avoid the possibly confusing overload of `tiff.crop(bounds, Array(1, 2))`

Also `targetBands` should be relaxed to `Seq[Int]` to allow using this method if `List` and `Range`. Also thats what we use in `subsetBands`